site/content/integrate/apps/bindings/_index.md

```diff
@@ -68,7 +68,7 @@ A leaf command must include:
 | `location`    | string | The label to use to define the command.                                                                                                      |
 | `hint`        | string | (Optional) Hint line on command autocomplete.                                                                                                |
 | `description` | string | (Optional) Description line on command autocomplete.                                                                                         |
-| `call`        | Call   | Call to perform when executing the command                                                                                                   |
+| `call`        | Call   | Call to perform when executing the command.                                                                                                   |
 | `form`        | Form   | (Optional) Form representing the parameters the command can receive. If no form is provided, a form call will be made to the specified call. |
 
 The call for these bindings will include in the context the user ID, the post ID, the root post ID if any, the channel ID and the team ID. It will also include the raw command.
```

Comment:
```suggestion
The context of the call for these bindings will include the user ID, the post ID, the root post ID (if any), the channel ID and the team ID. It will also include the raw command.
```